docs/section4/4-5.md

```diff
@@ -0,0 +1,38 @@
+# Airside Systems – Rule 4-5
+
+**Rule ID:** 4-5
+ 
+**Rule Description:** HVAC fans shall remain on during unoccupied hours in spaces that have health and safety mandated minimum ventilation requirements during unoccupied hours.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-4 Schedules exception #2 for the proposed building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. Applies to all HVAC systems that serve zones that have health and safety mandated minimum ventilation requirements during unoccupied hours. 
+ 
+**Function Call:** 
+
+1. get_zones_health_safety_vent_reqs()
+2. get_list_hvac_systems_associated_with_zone()
+
+**Manual Check:** None  
+  
+**Rule Logic:**
+**Applicability Check 1:** 
+- Get list of zones that are likely to have health and safety mandated minimum ventilation requirements during unoccupied hours: `applicable_zones_health_safety_unocc_list_p = get_zones_health_safety_vent_reqs(P_RMR)`
+- For each zone in applicable_zones_health_safety_unocc_list_p: `for zone_p in applicable_zones_health_safety_unocc_list_p:`
+    - Get list of hvac systems associated with this zone by calling the get_list_hvac_systems_associated_with_zone function: `zone_level_applicable_hvac_systems_list_p = get_list_hvac_systems_associated_with_zone(P_RMR,zone_p.id)`                          
+    - Add to master list of HVAC system that are applicable to this check: `applicable_hvac_systems_list_p = applicable_hvac_systems_list_p.append(zone_level_applicable_hvac_systems_list_p)`                    
+- Convert the list of applicable_hvac_systems_list_p to a set and the back to a list to eliminate duplicates after looping through all zones:`applicable_hvac_systems_list_p = list(set(applicable_hvac_systems_list_p))`                 
+- For each hvac_p in applicable_hvac_systems_list_p: `for each hvac_p in applicable_hvac_systems_list_p:`                         
+    - Get fan operation during unoccupied periods: `operation_during_unoccupied_p = hvac_p.fan_systems[0].operation_during_unoccupied`  
+    - Get minimum_outdoor_airflow: `minimum_outdoor_airflow_p = hvac_p.fan_systems[0].minimum_outdoor_airflow`  
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, if fan system operation during unoccupied periods is continuous in the P_RMR: `if operation_during_unoccupied_p == “CONTINUOUS” and minimum_outdoor_airflow_p > 0: PASS`
+    - Case 2: Else, for each applicable fan system,:`Else: UNDETERMINED and raise_message "THE HVAC SYSTEM SERVES ZONE(S) THAT APPEAR LIKELY TO HAVE HEALTH AND SAFETY MANDATED MINIMUM VENTILATION REQUIREMENTS DURING UNOCCUPIED HOURS AND THEREFORE MAY WARRANT CONTINUOUS OPERATION DURING UNOCCUPIED HOURS PER SECTION G3.1-4 SCHEDULES EXCEPTION #2 FOR THE PROPOSED BUILDING." `
+
```

Comment:
question for @mkarpman should case 2 be fail instead on undetermined?

Reply:
I agree.

Reply:
Okay it has been changed.